From f1b76698da3b13e40f657981018da9e3fa23b2c2 Mon Sep 17 00:00:00 2001
From: Christopher Albert <albert@tugraz.at>
Date: Thu, 30 Jul 2026 20:33:04 +0200
Subject: [PATCH] aarch64: Add NetBSD host CPU detection [PR124586]

Native AArch64 NetBSD builds omit driver-aarch64.o and fail to link
host_detect_local_cpu.  Add NetBSD to the AArch64 host pattern.

Fixes #11

Assisted-by: GPT-5.6-sol (OpenAI)

	PR target/124586

gcc/ChangeLog:

	* config.host (aarch64*-*-netbsd*): Add to the case pattern
	enabling driver-aarch64.o and host_detect_local_cpu.

Signed-off-by: Christopher Albert <albert@tugraz.at>
---
 gcc/config.host | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config.host b/gcc/config.host
index c9be68acb82..a4d10f28e69 100644
--- a/gcc/config.host
+++ b/gcc/config.host
@@ -100,7 +100,7 @@ esac
 
 case ${host} in
   aarch64*-*-freebsd* | aarch64*-*-linux* | aarch64*-*-fuchsia* |\
-  aarch64*-*-darwin*)
+  aarch64*-*-darwin* | aarch64*-*-netbsd*)
     case ${target} in
       aarch64*-*-*)
 	host_extra_gcc_objs="driver-aarch64.o"
-- 
2.55.0

